docs/guides/README.md

```diff
@@ -997,9 +997,299 @@ go run app.go
+```java
+#src/main/java/com/example/cosmicapp/controller/MainController.java
+package com.example.cosmicapp.controller;
+import com.example.cosmicapp.domain.Bucket;
+import com.example.cosmicapp.domain.Object;
+import com.example.cosmicapp.service.JsonParsingService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.env.Environment;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.util.UriComponentsBuilder;
+import java.util.LinkedHashMap;
+
+@Controller
+public class MainController {
+    @Autowired
+    private Environment env;
+    private static final String MAIN_PAGE = "main";
+    private static final String baseURL = "https://api.cosmicjs.com/v1/";
+    private static final String type = "posts";
+    private static final String props = "slug,title,content,metadata,";
+    @Autowired
+    JsonParsingService jsonParsingService;
+
+    @GetMapping
+    public String main(final Model model) {
+        String url = baseURL + env.getProperty("slug") + "/objects";
+        UriComponentsBuilder builder = UriComponentsBuilder.fromHttpUrl(url)
+                .queryParam("type", type)
+                .queryParam("read_key", env.getProperty("read_key"))
+                .queryParam("props", props);
+        LinkedHashMap<String, Object> objectsLinkedHashMap = (LinkedHashMap<String, Object>) jsonParsingService.parse(builder.toUriString());
+        ObjectMapper mapper = new ObjectMapper();
+        Bucket bucket = mapper.convertValue(objectsLinkedHashMap, Bucket.class);
+        model.addAttribute("objects", bucket.objects);
+        return MAIN_PAGE;
+    }
+}
+
+```
+
+### 8. Update Views
+Render your posts in the `src/main/resources/templates/main.html` with the following code:
+```html
+
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <title>List of Posts</title>
+
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css"></link>
+</head>
+<body>
+<h1> List of Posts</h1>
+<div class="container">
+    <div th:each="post : ${objects}">
+        <div th:if="${post.metadata != null && post.metadata.hero != null} ">
+            <img th:src="${post.metadata.hero.url}" width='500px' />
+        </div>
+        <div th:text="${post.title}"></div>
+        <br><br>
+    </div>
+</div>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+</body>
+</html>
+```
+
+### 9. Run your app
+Now you can run command below in your IntelliJ Terminal to import all the Maven dependencies:
```

Comment:
This is a little confusing. I think you can simply say:

### 9. Start your app
Start your Spring Boot application by running the command below, and go to http://localhost:8080. Dance 🎉
```bash
mvn spring-boot:run
```